include/manager.hpp

```diff
@@ -39,6 +40,117 @@ class Manager
      */
     ~Manager() = default;
 
+    /**
+     * @brief Update keyword value.
+     *
+     * API can be used to update VPD keyword.
+     *
+     * To update a keyword on cache, input path should be d-bus object path.
+     * Eg: ("/xyz/openbmc_project/inventory/system/chassis/motherboard").
+     *
+     * To update a keyword on hardware, input path should be EEPROM path.
+     * Eg:
+     * ("/sys/bus/i2c/drivers/at24/9-0052/eeprom").
+     *
+     * To update a keyword on both cache and hardware, input path should be
+     * d-bus object path. Eg:
```

Comment:
Same here D-Bus

Reply:
Please check the same at other places as well.

Reply:
done